pandas_sphinx_theme/layout.html

```diff
@@ -1,6 +1,17 @@
 
 {% extends "basic/layout.html" %}
 
+{# prev/next buttons #}
+{% macro prev_next(prev, next, prev_title='', next_title='') %}
+ {%- if prev %}
+    <a class='left-prev' id="prev-link" href="{{ prev.link|e }}" title="{{ _('previous page')}}">{{ prev_title or prev.title }}</a>
+ {%- endif %}
+ {%- if next %}
+    <a class='right-next' id="next-link" href="{{ next.link|e }}" title="{{ _('next page')}}">{{ next_title or next.title }}</a>
+ {%- endif %}
+ <div style='clear:both;'></div>
```

Comment:
Do you know what this is for? Add a comment?

Reply:
I think it's a carry-over from the Jupyter templates, deleted it :-)